packages/adblocker/src/resources.ts

```diff
@@ -297,17 +297,33 @@ export default class Resources {
     }
   }
 
-  public getResource(name: string): { body: string; contentType: string; dataUrl: string } {
-    const { body, contentType } = this.resourcesByName.get(name) || getResourceForMime(name);
+  public getResource(name: string): {
+    filename: string;
+    body: string;
+    contentType: string;
+    dataUrl: string;
+  } {
+    let resource:
+      | {
+          name: string;
+          contentType: string;
+          body: string;
+        }
+      | undefined = this.resourcesByName.get(name);
+    if (resource === undefined) {
+      const extensionIndex = name.lastIndexOf('.');
```

Comment:
It might not happen but is it safe to always take the last index rather than the first index? (e.g. `foo.tar.bz2`)

Reply:
In that pattern, I guess the file is compressed tar stream by bz2, which involves two steps. I believe it's better to stick with bz2.